src/test/channelFilterRouteSpec.js

```diff
@@ -0,0 +1,44 @@
+var routes = [
+  'content/search',
+  'course/search',
+  'framework/category/search',
+  'framework/term/search',
+  'search'
+]
+
+var requestMiddleware = require('../middlewares/request.middleware')
+var httpMocks = require('node-mocks-http')
+
+var baseUrl = 'http://localhost:5000/v1/'
+var async = require('async')
+
+describe('Check', function () {
+  async.forEach(routes, function (route, callback) {
+    describe('Composite search services', function () {
```

Comment:
the name of this test needs to be improved to indicate what the test scenario is